### PR TITLE
Added MangaRuhu extension for Turkish manga source. 

### LINKED ISSUE #5753: Manga Ruhu
### Source name

Mangaruhu

### Source link

https://mangaruhu.com/

### Source language

Turkish

### Other details

_No response_

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Forza06**
Closes [#5753](https://github.com/keiyoushi/extensions-source/issues/5753)

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/tr/mangaruhu/build.gradle [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ext {
+    extName = 'Manga Ruhu'
+    extClass = '.MangaRuhu'
+    themePkg = 'madara'
+    baseUrl = 'https://mangaruhu.com'
+    overrideVersionCode = 1
```
Comment (vetleledaal) [inline src/tr/mangaruhu/build.gradle]:
```suggestion
    overrideVersionCode = 0
```

File: src/tr/mangaruhu/res/mipmap-xxxhdpi/ic_launcher.png
Comment (vetleledaal) [inline src/tr/mangaruhu/res/mipmap-xxxhdpi/ic_launcher.png:1]: Please use the icon generator mentioned in the [contributing guidelines](https://github.com/keiyoushi/extensions-source/blob/6577a20ead5ab7896aa5349602652c38bf657e4f/CONTRIBUTING.md#tools).

File: src/tr/mangaruhu/src/eu/kanade/tachiyomi/extension/tr/mangaruhu/MangaRuhu.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class MangaRuhu : Madara(
+    "Manga Ruhu",
+    "https://mangaruhu.com",
+    "tr",
+    SimpleDateFormat("dd/MM/yyyy", Locale.ENGLISH),
```
Comment (vetleledaal) [inline src/tr/mangaruhu/src/eu/kanade/tachiyomi/extension/tr/mangaruhu/MangaRuhu.kt]: Date format doesn't seem to match site. Is the API returning a different format?

Try:
```suggestion
    SimpleDateFormat("d MMMM yyyy", Locale("tr")),
```

**Review by vetleledaal — CHANGES_REQUESTED** [review]

File: src/tr/mangaruhu/res/mipmap-xxxhdpi/ic_launcher.png
Comment (Forza06) [reply src/tr/mangaruhu/res/mipmap-xxxhdpi/ic_launcher.png:1]: I already used the icon generator in contributing guidelines I just changed the Shape value to none

Comment (AwkwardPeak7) [reply src/tr/mangaruhu/res/mipmap-xxxhdpi/ic_launcher.png:1]: icon shape should be square with rounded corners

Comment (Forza06) [reply src/tr/mangaruhu/res/mipmap-xxxhdpi/ic_launcher.png:1]: I changed the icons to square

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**